doc/build/orm/session_basics.rst

```diff
@@ -522,7 +522,11 @@ ways to refresh its contents with new data from the current transaction:
   and indicates that it should return objects that are unconditionally
   re-populated from their contents in the database::
 
-    u2 = session.query(User).populate_existing().filter(id=5).first()
+    u2 = session.scalars(
+        select(User)
+        .where(User.id == 5)
+        .execution_options(populate_existing=True)
+    ).first()
```

Comment:
**mike bayer** (zzzeek) wrote:

let's use one() here

View this in Gerrit at https://gerrit.sqlalchemy.org/c/sqlalchemy/sqlalchemy/+/3729